### PR TITLE
MASS FIXER!!!

### PR DESCRIPTION
**Pull request by ProfessorPopoff**
## About The Pull Request

This PR fixes a few things, and adds some more soul.

PA sets now protect you from fall damage, while assuring others won't live much longer...

https://i.gyazo.com/5e60e1440c2605cbad4850324c94cd28.gif

Red siding walls fixed

Body thuds

More SOUNDS...

Barony map fixes

## Why It's Good For The Game

good soul is good

### CODE REVIEW

**Comment by ProfessorPopoff** [comment]
i hit enter :DDDDD

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -49,20 +49,61 @@
 		message = FALSE
 	return ..()
 
+// MOJAVE SUN EDIT BEGIN
 /mob/living/proc/ZImpactDamage(turf/T, levels)
 	if(SEND_SIGNAL(src, COMSIG_LIVING_Z_IMPACT, levels, T) & NO_Z_IMPACT_DAMAGE)
 		return
+	var/mob/living/carbon/human/savage = src // Power armour landing interactions - Fall damage negation / general chaos creation
+	if(istype(savage.wear_suit, /obj/item/clothing/suit/space/hardsuit/ms13/power_armor)) // If they're wearing PA, it's time to go sicko mode and start doing some crazy shit.
```
Comment (ChungusGamer666) [inline code/modules/mob/living/living.dm:57]: USE A SIGNAL CHUD!!!!!!!!!!!!!

Comment (ProfessorPopoff) [reply code/modules/mob/living/living.dm:57]: like... a traffic signal...?

Comment (ProfessorPopoff) [reply code/modules/mob/living/living.dm:57]: I don't have time to do it before the test and I'll fix it later :)

Comment (Koshenko) [reply code/modules/mob/living/living.dm:57]: Will need to refactor this.  Will merge for now for the tests.

**Pull request merged**